ajax/libs/jquery-cookiebar/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-cookiebar",
+  "filename": "jquery.cookiebar.min.js",
+  "version": "1.0.5",
+  "description": "jQuery CookieBar Plugin",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/missing-code/jquery-cookiebar.git"
+  },
+  "keywords": [
+    "cookie",
+    "cookiebar",
+    "jquery",
+    "cookielaw",
+    "eucookies"
+  ],
+  "author": "Stefan Kecskes <info@skey.uk> (https://skey.uk)",
+  "license": "MIT",
+  "homepage": "http://www.primebox.co.uk/projects/cookie-bar/",
+  "npmName": "jquery-cookiebar",
```

Comment:
@hare1039 thanks! I revised package.json. @PeterDaveHello Please help me review this!